cumulus/parachains/runtimes/assets/asset-hub-rococo/Cargo.toml

```diff
@@ -7,74 +7,86 @@ description = "Rococo variant of Asset Hub parachain runtime"
 license = "Apache-2.0"
 
 [dependencies]
-codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = ["derive", "max-encoded-len"] }
+codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = [
+	"derive",
+	"max-encoded-len",
```

Comment:
What is this style 🤔 ?

Reply:
Default settings of this linter https://taplo.tamasfe.dev/

Reply:
Can turn off the multi-line wrapping if you prefer :P

Reply:
Yea, i think we should keep the crates consistent.